### PR TITLE
[1.16] bump conmon to 2.0.8

### PR DESCRIPTION
**Pull request by haircommander**
As well as implement versioning scheme for circle images, where we push to \<branch>-\<sem-version>, as opposed to \<sem-version>.

Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -74,7 +74,7 @@ RUN VERSION=v0.8.1 &&\
         | tar xfz - -C /opt/cni/bin
 
 # Install conmon
-RUN VERSION=v2.0.5 &&\
+RUN VERSION=v2.0.6 &&\
```
Comment (mrunalp) [inline Dockerfile]: 2.0.7

Comment (haircommander) [reply Dockerfile]: fixed

**Comment by haircommander** [comment]
needs: https://github.com/cri-o/cri-o/pull/3046

**Pull request merged**